release/v1alpha1/types.go

```diff
@@ -0,0 +1,95 @@
+package v1alpha1
+
+import metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+// +genclient
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// ReleasePayload aggregates the outputs of the promotion tests
```

Comment:
Update the comment.  Be sure to include what information is known at creation time and which actor creates this resource and when they create it.